perception_launch/package.xml

```diff
@@ -18,6 +18,8 @@
   <exec_depend>naive_path_prediction</exec_depend>
   <exec_depend>roi_cluster_fusion</exec_depend>
   <exec_depend>shape_estimation</exec_depend>
+  <exec_depend>tensorrt_yolo</exec_depend>
+  <exec_depend>object_range_splitter</exec_depend>
```

Comment:
missing object_merger

Reply:
Fixed in https://github.com/tier4/autoware_launcher.iv.universe/pull/56/commits/cd9edd01c30f959512cf4b1ab816164a3eebb34e